be2-scala/src/test/scala/ch/epfl/pop/model/network/method/message/data/socialMedia/AddChirpSuite.scala

```diff
@@ -28,7 +29,7 @@ class AddChirpSuite extends FunSuite with Matchers {
 object AddChirpExample {
   final val TEXT: String = "text"
   final val PARENT_ID: Option[String] = None
-  final val TIMESTAMP = Timestamp(0)
+  final val TIMESTAMP: Timestamp = Timestamp(0)
```

Comment:
I think at some places it was unnecessary to add the typing

Reply:
I personally don't mind but the "migrate" plugin made that change.